dev-aws/kafka-shared-msk/iam/iam.tf

```diff
@@ -292,3 +292,9 @@ module "castle_processor" {
   consume_topics   = [(kafka_topic.iam_credentials_v1.name)]
   consume_groups   = ["iam.castle-processor"]
 }
+
+module "cbc_fraud_detection_consumer" {
+  source           = "../../../modules/tls-app"
+  consume_topics   = [kafka_topic.iam_credentials_v1_public.name]
+  cert_common_name = "cbc/cbc-fraud-detection-consumer"
+}
```

Comment:
does this also need a consume group?

Reply:
Indeed, I also checked this, and they use the same consumer group for multiple topics: https://kafka-ui.dev.uw.systems/ui/clusters/shared-msk/consumer-groups/cbc.cbc-fraud-detection-consumer-v1
The ACL for it needs to be defined in a single place

Reply:
thanks, make sense 👍